.github/workflows/release.yml

```diff
@@ -44,11 +44,13 @@ jobs:
         run: |
           pip install --constraint=.github/workflows/constraints.txt ansible
 
-      - name: Trigger a new import on Galaxy.
+      - name: Release to Ansible Galaxy.
         run: >-
-          ansible-galaxy role import
-          --api-key ${{ secrets.GALAXY_TOKEN }} $(echo ${{ github.repository }}
-          | cut -d/ -f1) $(echo ${{ github.repository }} | cut -d/ -f2)
+          ansible-playbook -i 'localhost,' galaxy-deploy.yml
+          -e "namespace=$(echo ${{ github.repository }}
+          | cut -d/ -f1)
+          -e "collection"=$(echo ${{ github.repository }}
+          | cut -d/ -f2)
```

Comment:
looks like it's missing 
```bash
#   -e "github_tag=${{ github.ref }}"
```
as it is used in various places in the playbooks

also this step was run everytime a merge was done on master.
so It may create issues to push branch master and tags everytime on galaxy.
Shouldn't we just to galaxy only when adding a new tag ?


Well As long as #209 is not merged we should be able to test that in my namespace

Reply:
Good catch, fixed.